apps/docs/pages/guides/platform/backups.mdx

```diff
@@ -66,7 +78,7 @@ By default, WAL files are backed up at two minute intervals. If these files cros
 
 ![PITR dashboard](/docs/img/backups-pitr-dashboard.png)
 
-You can access PITR in the [Point in Time](https://app.supabase.com/project/_/database/backups/pitr) settings in the Dashboard. The recovery period of a project is indicated by the earliest and latest points of recoveries displayed in one’s preferred timezone. If need be, the maximum amount of this recovery period can be modified accordingly.
+You can access PITR in the [Point in Time](https://app.supabase.com/project/_/database/backups/pitr) settings in the Dashboard. The recovery period of a project is indicated by the earliest and latest points of recoveries displayed in one's preferred timezone. If need be, the maximum amount of this recovery period can be modified accordingly.
```

Comment:
side note: we should avoid terms like "one's". 

If you wouldn't speak it, we shouldn't write it.

Reply:
Thank you - updated.